mumerge.py

```diff
@@ -219,12 +231,14 @@ def inputs_processor():
             samples = []
             for line in f:
                 samp = tuple(line.strip().split("\t"))
-                assert len(samp) == 3, "Sample line must contain three fields."
+                assert len(samp) >= 3, "Sample line must contain three fields."
                 samples.append(samp)
             
             # Assign bedfiles and sampids (input must have 3 columns)
-            bedfiles, sampids, groups = zip(*samples)
-            
+            if len(samp) > 0:
```

Comment:
`samp` is an individual line in the input file. It's length is the number of columns in the input file. Why are you checking whether or not it's greater than zero?